src/pages/steps/ContactInfoEntry.tsx

```diff
@@ -120,6 +123,9 @@ const Form = ({
   contactInfo,
   onAddContactInfo,
   onAddBookingInfo,
+  onAddOrganizerContactInfo,
+  isOrganizer = false,
+  withReservationInfo = false,
```

Comment:
These defaultValues would be better as defaultProps to the Form components, to be consistent